### ISSUE TITLE
Add Twitter list timeline with important accounts

### ISSUE THREAD

**Issue by TonyToscana** [GuilleAmutio/sos-valencia#2]
Create a twitter list with accounts that have important announcements and information (112 CV, DGT...) and embed into the site

**Comment by GuilleAmutio** [comment]
Yes, indeed, having another page "Noticias" o "En vivo" with a red dot by its side would be very useful.

Twitter/X accounts to follow:

- https://x.com/AEMET_CValencia AEMET
- https://x.com/BombersValencia Bombers
- https://x.com/AjuntamentVLC Ajuntament
- https://x.com/generalitat Generalitat
- https://x.com/GVA112 Emergencies 112
- https://x.com/policialocalvlc Policia Local
- https://x.com/levante_emv Levante EMV
- https://x.com/GVAsanitat Sanitat
- https://x.com/apunt_media Apunt

This navbar should be sticky and above the search bar, font ubuntu, highlight in red. Navbar would contain "Inicio", where all the posts are, and "🔴En vivo" , where the tweets are displazed from latest to oldest.

This better be tested on local before deploying. I can not tackle it until Sunday, so feel free to tackle it
